client-v2/src/shared/selectors/collectionItem.ts

```diff
@@ -17,33 +17,6 @@ const createSelectCollectionItemType = () =>
     }
   );
 
-const createSelectActiveImageUrl = () =>
```

Comment:
This was only being used in one place.... and it was creating problems because it occasionally produced different results than the simple `externalArticle.fields.thumbnail`. Articles with overridden images behaved in unpredictable ways.